.github/workflows/ci.yml

```diff
@@ -130,5 +130,11 @@ jobs:
           ZULIP_API_TOKEN: ${{ secrets.ZULIP_API_TOKEN }}
         run: |
           ~/.local/bin/zulip-send --stream miri --subject "Cron Job Failure" \
-            --message 'Dear @**RalfJ** and @**oli**\n\nIt would appear that the Miri cron job build failed. Would you mind investigating this issue?\n\nThanks in advance!\nSincerely,\nThe Miri Cronjobs Bot' \
+            --message 'Dear @**RalfJ** and @**oli**
+
+          It would appear that the Miri cron job build failed. Would you mind investigating this issue?
```

Comment:
I think this will make the indentation part of the message, won't it?

Reply:
The better fix might be to use `$(echo -e 'Message\nwith\nnewlines')`

Reply:
> I think this will make the indentation part of the message, won't it?

No. I [tried](https://github.com/hyd-dev/github-actions/blob/17d960be3df7d4f714533bea0b53ea32d3c82631/cargo-auto-update/action.yml#L32-L36) something like that before on GitHub Actions, and it produced [a message without indentation](https://github.com/hyd-dev/github-actions/runs/1716793883#step:3:243) (the "Update" line is aligned with "Auto update").

Reply:
But that was with a `"` quote, not `'`, which might make a difference.

Reply:
Oh, the point is that the only indentation is yaml indentation, and that one is stripped. I see.